### PR TITLE
Updating changelog

### PR DESCRIPTION
**Pull request by kcoleman731**
Updating Changelog for PR https://github.com/CocoaPods/CocoaPods/pull/4341

### CODE REVIEW

**Comment by segiddins** [comment]
Needs to go in the master section

**Comment by kcoleman731** [comment]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,7 +24,6 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Marius Rackwitz](https://github.com/mrackwitz)
   [#4137](https://github.com/CocoaPods/CocoaPods/pull/4137)
 
-
```
Comment (kylef) [inline CHANGELOG.md]: Please don't remove this, this is an intentional double new line between releases.

Comment (kcoleman731) [inline CHANGELOG.md]: :+1: My mistake. Thnx @kylef

**Pull request merged**